.github/workflows/javaapi.yml

```diff
@@ -0,0 +1,123 @@
+name: javaapi
+
+on:
+  push:
+    paths-ignore:
+      - "tools/**"
+
+  pull_request:
+    paths-ignore:
+      - "tools/**"
+      - "docs/**"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test_javaapi:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.7]
+        torch: [1.9.0]
+        mmcv: [1.4.5]
+        include:
+          - torch: 1.9.0
+            torch_version: torch1.9
+            torchvision: 0.10.0
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install unittest dependencies
+        run: |
+          pip install cmake onnx
+      - name: update
+        run: sudo apt update
+      - name: gcc-multilib
+        run: sudo apt install gcc-multilib g++-multilib wget libprotobuf-dev protobuf-compiler
+      - name: Install ncnn
```

Comment:
`ncnn` is not necessary for test Java API. `onnxruntime` is fair enough, which by the way has prebuilt package. We don't need to build onnxruntime at all